### PR TITLE
Hyundai CAN-FD: Lead Relative Speed

### PR DESCRIPTION
**Pull request by sunnyhaibin**
Properly defines the signal for the relative speed to the tracked lead car. This has been confirmed on camera-based SCC CAN-FD platforms.

Segment ID in screenshot: `5a681a11ff756d93|2023-11-18--10-35-23--0`

image

image

### CODE REVIEW

File: hyundai_canfd.dbc [hunk 1]
```diff
@@ -263,7 +263,7 @@ BO_ 416 SCC_CONTROL: 32 ADRV
  SG_ aReqValue : 128|11@1+ (0.01,-10.23) [-10.23|10.24] "m/s^2" XXX
  SG_ ZEROS_7 : 63|8@0+ (1,0) [0|255] "" XXX
  SG_ ACCMode : 68|3@1+ (1,0) [0|7] "" XXX
- SG_ NEW_SIGNAL_12 : 35|9@1+ (0.1,0) [0|255] "" XXX
+ SG_ ACC_ObjRelSpd : 35|9@1+ (0.1,-16.4) [0|255] "" XXX
```
Comment (sshane) [inline hyundai_canfd.dbc:266]: can you add units and update min/max?

Comment (sunnyhaibin) [reply hyundai_canfd.dbc]: @sshane Updated!

**Comment by sshane** [comment]
Looks like it matches, it just lags behind by a little bit (found by plotting wheel speeds with stationary lead)

image


image

File: hyundai_canfd.dbc [hunk 2]
```diff
@@ -263,7 +263,7 @@ BO_ 416 SCC_CONTROL: 32 ADRV
  SG_ aReqValue : 128|11@1+ (0.01,-10.23) [-10.23|10.24] "m/s^2" XXX
  SG_ ZEROS_7 : 63|8@0+ (1,0) [0|255] "" XXX
  SG_ ACCMode : 68|3@1+ (1,0) [0|7] "" XXX
- SG_ NEW_SIGNAL_12 : 35|9@1+ (0.1,0) [0|255] "" XXX
+ SG_ ACC_ObjRelSpd : 35|9@1+ (0.1,-16.4) [-16.4|34.8] "m/s" XXX
```
Comment (sshane) [inline hyundai_canfd.dbc:266]:
```suggestion
 SG_ ACC_ObjRelSpd : 35|9@1+ (0.1,-16.4) [-16.4|34.7] "m/s" XXX
```

**Pull request merged**